src/plugins/iLoveSpam/index.ts

```diff
@@ -27,7 +27,7 @@ export default definePlugin({
         {
             find: "hasFlag:{writable",
             replacement: {
-                match: /if\((\i)<=(?:1<<30|1073741824)\)return/,
+                match: /if\((\i)<=(?:0x40000000|1073741824)\)return/,
```

Comment:
we should just have both (I added extra group so it's more clear it's the same flag)

```suggestion
                match: /if\((\i)<=(?:(?:0x40000000|1<<30)|1073741824)\)return/,
```